openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -154,6 +155,13 @@ def get_schedules_with_target_date_by_bin_and_orgs(
 
         schedules = self.filter_by_org(schedules)
 
+        try:
+            # .. filter_implemented_name: ScheduleQuerySetRequested
+            # .. filter_type: org.openedx.learning.schedule.queryset.requested.v1
+            schedules = ScheduleQuerySetRequested.run_filter(schedules)
+        except ScheduleQuerySetRequested.PreventScheduleQuerysetRequest as exc:
+            schedules = exc.schedules
```

Comment:
Updated:
Reading https://github.com/openedx/openedx-filters/pull/235#discussion_r1880746425
I think we can use that exception if we want to prevent the schedules, but if not, it's better if we handle the default flow ("the same QuerySet is returned and the command execution continues") from the pipeline.